pyteal/ast/ecdsa_test.py

```diff
@@ -26,22 +26,30 @@ def test_ecdsa_decompress():
         ]
     )
 
-    actual, _ = pubkey.__teal__(teal5Options)
+    actual, _ = pubkey.__teal__(teal7Options)
```

Comment:
@jdtzmn Let me know if the thinking makes sense or if you'd like to discuss live?

* I like the idea to parameterize the test though I think the old assertion against `teal5Options` must be retained for _Secp256k1_.
* As shown, it's possible to bump `Secp256k1.min_version` to 7 without the test failing.
* If we agree a gap exists, I think there's at least several ways to address it.
  * A local solution is to either branch the test based on the opcode or build a dictionary from version to options.
  * Eventually, I think a more systemic solution will be needed to vet _all_ versions.  But, for now, I think it's _safe_ to solve the issue local to `ecdsa_test.py`.